src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -2427,26 +2423,48 @@ static JsonObject nod(UUID holdingsRecordId) {
     return nod(UUID.randomUUID(), holdingsRecordId);
   }
 
+  static JsonObject nodWithNoBarcode(UUID holdingsRecordId) {
+    return removeBarcode(nod(holdingsRecordId));
+  }
+
   private static JsonObject uprooted(UUID itemId, UUID holdingsRecordId) {
     return createItemRequest(itemId, holdingsRecordId, "657670342075");
   }
 
+  private static JsonObject uprooted(UUID holdingsRecordId) {
+    return uprooted(UUID.randomUUID(), holdingsRecordId);
+  }
+
   private static JsonObject temeraire(UUID itemId, UUID holdingsRecordId) {
     return createItemRequest(itemId, holdingsRecordId, "232142443432");
   }
 
+  private static JsonObject temeraire(UUID holdingsRecordId) {
+    return temeraire(UUID.randomUUID(), holdingsRecordId);
+  }
+
   private static JsonObject interestingTimes(UUID itemId, UUID holdingsRecordId) {
     return createItemRequest(itemId, holdingsRecordId, "56454543534");
   }
 
+  private static JsonObject interestingTimes(UUID holdingsRecordId) {
+    return interestingTimes(UUID.randomUUID(), holdingsRecordId);
+  }
+
+  static JsonObject removeBarcode(JsonObject item) {
+    item.remove("barcode");
+    return item;
+  }
+
   private Items findItems(String searchQuery) throws Exception {
     CompletableFuture<Response> searchCompleted = new CompletableFuture<>();
 
     client.get(itemsStorageUrl("?query=") + urlEncode(searchQuery),
       StorageTestSuite.TENANT_ID, ResponseHandler.json(searchCompleted));
 
-    return searchCompleted.get(5, TimeUnit.SECONDS).getJson()
-      .mapTo(Items.class);
+    Response response = searchCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(response.getStatusCode(), is(200));
+    return response.getJson().mapTo(Items.class);
```

Comment:
I find it easier to notice assertions when they are separated from the statements around them

```suggestion
    Response response = searchCompleted.get(5, TimeUnit.SECONDS);
    
    assertThat(response.getStatusCode(), is(200));
    
    return response.getJson().mapTo(Items.class);
```